src/ray/common/ray_config_def.h

```diff
@@ -262,6 +262,9 @@ RAY_CONFIG(int64_t, internal_gcs_service_connect_wait_milliseconds, 100)
 /// When this happens, gcs server will kill itself.
 RAY_CONFIG(int64_t, gcs_redis_heartbeat_interval_milliseconds, 100)
 
+/// Maximum number of keys in one batch to scan from storage.
+RAY_CONFIG(uint32_t, gcs_service_scan_batch_size, 1000)
```

Comment:
This is not used? Also, I think this is specific to redis, and better name it `gcs_service_redis_scan_batch_size`.

Reply:
sounds good.